<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="vertSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QSplitter" name="horzSplitter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>180</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>3</number>
       </property>
       <widget class="DbTree" name="tree"/>
       <widget class="QTabWidget" name="mainTab">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Data</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="dataTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Schema</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="schemaTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Index</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="indexTable"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>SQL Query</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="queryEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>&amp;Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="submitButton">
            <property name="text">
             <string>&amp;Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Add_Database"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Add_Database">
   <property name="text">
    <string>&amp;Add Database</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="toolTip">
    <string>Insert Row</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="text">
    <string>&amp;Delete Row</string>
   </property>
   <property name="toolTip">
    <string>Delete Row</string>
   </property>
  </action>
  <action name="fieldStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit on &amp;Field Change</string>
   </property>
   <property name="toolTip">
    <string>Commit on Field Change</string>
   </property>
  </action>
  <action name="rowStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit on &amp;Row Change</string>
   </property>
   <property name="toolTip">
    <string>Commit on Row Change</string>
   </property>
  </action>
  <action name="manualStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit &amp;Manually</string>
   </property>
   <property name="toolTip">
    <string>Commit Manually</string>
   </property>
  </action>
  <action name="submitAction">
   <property name="text">
    <string>&amp;Submit All</string>
   </property>
   <property name="toolTip">
    <string>Submit Changes</string>
   </property>
  </action>
  <action name="revertAction">
   <property name="text">
    <string>&amp;Revert All</string>
   </property>
   <property name="toolTip">
    <string>Revert</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="text">
    <string>S&amp;elect</string>
   </property>
   <property name="toolTip">
    <string>Refresh Data from Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DbTree</class>
   <extends>QTreeView</extends>
   <header>dbtree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
